.gitignore

```diff
@@ -87,3 +87,5 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+src/nv_midbox_websock/templates/static/bower_components/
```

Comment:
Do not add anything in the gitignore.
Its not a good practice.
Either keep those files in the source tree or provide option to install it.